### PR TITLE
added expiration to token

### PR DESCRIPTION
**Pull request by khodzha**
hi

right now its impossible to specify exp in tokens

this pr adds expires() method to Token and serializes that if present

### CODE REVIEW

**Comment by bbigras** [comment]
Any progress on this?

File: src/lib.rs [hunk 1]
```diff
@@ -86,6 +86,10 @@ pub struct Claims {
 
     /// NATS claims
     pub nats: NatsClaims,
+
+    /// Time when the token expires (in seconds since the unix epoch)
+    #[serde(rename = "exp", skip_serializing_if = "Option::is_none")]
+    pub expires: Option<i64>,
```
Comment (jamesbirtles) [inline src/lib.rs:92]: is there any reason for this to be an i64 instead of u64?

**Review by jamesbirtles — COMMENTED** [review]
Oops sorry for the delay, I didn't have notifications on for this repo!

File: src/lib.rs (hunk 1 above)
Comment (jamesbirtles) [reply src/lib.rs:92]: Looks like the official go implementation uses i64 as well
https://github.com/nats-io/jwt/blob/4a8a7328f44f94e4e9d4582fbd7f439100a2cd0e/claims.go#L66

I suppose that means issued_at should probably be changed to an i64 too at some point.

For now I'll merge this as is

**Review by jamesbirtles — APPROVED** [review]

**Comment by jamesbirtles** [comment]
Published in 0.2.0 🎉

**Pull request merged**